src/main/java/org/folio/mosaic/service/MosaicOrderConverter.java

```diff
@@ -0,0 +1,296 @@
+
+    return new CompositePurchaseOrder()
+      .withTemplate(template.getId())
+      .withOrderType(orderType)
+      .withAcqUnitIds(template.getAcqUnitIds())
+      .withBillTo(template.getBillTo())
+      .withShipTo(template.getShipTo())
+      .withVendor(template.getVendor())
+      .withNotes(template.getNotes())
+      .withCustomFields(template.getCustomFields())
+      .withPoLines(templatePoLines);
+  }
+
+  /**
+   * Applies overrides from the request to the purchase order.
+   * This will set values based on the request, overriding any template values.
+   *
+   * @param order       The purchase order to modify
+   * @param mosaicOrder The request containing override values
+   */
+  private void applyOverrides(CompositePurchaseOrder order, MosaicOrder mosaicOrder) {
+    if (mosaicOrder.getVendor() != null) {
+      order.setVendor(mosaicOrder.getVendor());
+    }
+    if (mosaicOrder.getBillTo() != null) {
+      order.setBillTo(mosaicOrder.getBillTo());
+    }
+    if (mosaicOrder.getShipTo() != null) {
+      order.setShipTo(mosaicOrder.getShipTo());
+    }
+
+    if (CollectionUtils.isNotEmpty(mosaicOrder.getAcqUnitIds())) {
+      order.setAcqUnitIds(mosaicOrder.getAcqUnitIds());
+    }
+
+    if (mosaicOrder.getCustomFields() != null) {
+      var convertedCustomFields = new CustomFields();
+      mosaicOrder.getCustomFields().getAdditionalProperties().forEach(convertedCustomFields::withAdditionalProperty);
+      order.setCustomFields(convertedCustomFields);
+    }
+
+    var poLine = new PoLine();
+
+    if (isNotBlank(mosaicOrder.getTitle())) {
+      poLine.setTitleOrPackage(mosaicOrder.getTitle());
+    }
+
+    updatePoLineCost(mosaicOrder, poLine);
+
+    updatePoLineContributors(mosaicOrder, poLine);
+
+    if (isNotBlank(mosaicOrder.getPublicationDate())) {
+      poLine.setPublicationDate(mosaicOrder.getPublicationDate());
+    }
+
+    if (isNotBlank(mosaicOrder.getEdition())) {
+      poLine.setEdition(mosaicOrder.getEdition());
+    }
+
+    updatePoLineDetails(mosaicOrder, poLine);
+
+    if (isNotBlank(mosaicOrder.getReceivingNote())) {
+      var details = poLine.getDetails() != null ? poLine.getDetails() : new Details();
+      details.setReceivingNote(mosaicOrder.getReceivingNote());
+      poLine.setDetails(details);
+    }
+
+    updatePoLineVendor(mosaicOrder, poLine);
+
+    if (isNotBlank(mosaicOrder.getUserLimit())) {
+      var eresource = poLine.getEresource() != null ? poLine.getEresource() : new Eresource();
+      eresource.setUserLimit(mosaicOrder.getUserLimit());
+      poLine.setEresource(eresource);
+    }
+
+    if (isNotBlank(mosaicOrder.getRequesterName())) {
+      poLine.setRequester(mosaicOrder.getRequesterName());
+    }
+
+    if (isNotBlank(mosaicOrder.getSelectorName())) {
+      poLine.setSelector(mosaicOrder.getSelectorName());
+    }
+
+    if (isNotBlank(mosaicOrder.getPoLineNote())) {
+      order.setNotes(singletonList(mosaicOrder.getPoLineNote()));
```

Comment:
we set notes for order, not for po line, so need to rename this field polineNote
https://github.com/folio-org/acq-models/blob/master/mod-orders/schemas/composite_purchase_order.json#L47

Reply:
changed to orderNote